cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -228,6 +233,7 @@ func upgradeComponent(component string, waiter apiclient.Waiter, pathMgr StaticP
 
 	if waitForComponentRestart {
 		fmt.Println("[upgrade/staticpods] Waiting for the kubelet to restart the component")
+		fmt.Printf("[upgrade/staticpods] This operation will timeout in %v\n", UpgradeManifestTimeout)
```

Comment:
why not just alter message above to add timeout value ?

Reply:
@kad updated